Cargo.toml

```diff
@@ -182,7 +182,7 @@ cidr-utils = "0.5.0"
 pin-project = "1.0.1"
 nats = { version = "0.8.6", optional = true }
 k8s-openapi = { version = "0.9", features = ["v1_16"], optional = true }
-portpicker = "0.1.0"
+portpicker = { version = "0.1.0", git = "https://github.com/timberio/portpicker-rs", rev = "d15829e906516720881584ff3301a0dd04218fbb" }
```

Comment:
Can we create an upstream PR for this (and others)? I know some crates are unmaintained and unlikely to see any action, but it's still a good practice to follow.

Reply:
PR already exists: https://github.com/Dentosal/portpicker-rs/pull/3